sphinx6-preparations.rst

```diff
@@ -0,0 +1,37 @@
+.. post:: Dec 5, 2022
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Sphinx 6 is coming soon. Here are our thoughts about preparations that we
+      are making, which may affect projects looking to upgrade.
+
+
+Sphinx 6 is coming
+==================
+
+Sphinx 6 is not yet released, but it may be released before the arrival of our next newsletter.
+Here are some of our considerations for getting ready for Sphinx 6:
+
+- Sphinx 6 focuses on reducing the maintenance burden for the future,
```

Comment:
We should break out the breaking changes at least to highlight them really explicitly.

Reply:
This is fixed now, moved all "motivational" notes to a final outro.